src/components/shared/LanguageSelector/LanguageSelector.js

```diff
@@ -9,29 +9,52 @@ import { injectIntl } from 'react-intl';
  */
 const LanguageSelector = ({ intl }) => {
   const lang = intl.locale;
+  const languageDropdown = useRef(null);
+
+  useEffect(() => {
+    const menubutton = new window.Menubutton(languageDropdown.current);
```

Comment:
maybe can add a comment with mention "MenuButton" is define in UI Library

Reply:
That would be a good idea 👍